lib/facter/custom_facts/core/execution.rb

```diff
@@ -96,22 +96,25 @@ def exec(command)
       end
 
       # Execute a command and return the output of that program.
+      #
       # @param command [String] Command to run
       #
       # @param options [Hash] Hash with options for the command
-      #
-      # Options accepted values :on_fail How to behave when the command could
+      # @option options [Symbol] :on_fail How to behave when the command could
```

Comment:
Can someone who knows ruby confirm that `Symbol` is actually the correct type here?

Reply:
https://github.com/puppetlabs/facter/blob/main/lib/facter/custom_facts/core/execution/base.rb#L114